reanahub/reana
# helm: generalize disk quota cronjob

* cover also CPU updates
* replace `disk_update` with `periodic_update_policy`

closes https://github.com/reanahub/reana/issues/605

#### To test:

To check multiple scenarios, edit `values-dev.yaml` to use old or new values (backward compatibility). E.g. 
- Only calculate CPU quota on workflow termination
- Run periodic quota updater every two minutes

```diff
diff --git a/helm/configurations/values-dev.yaml b/helm/configurations/values-dev.yaml
index 01aa44d..c0dcc11 100644
--- a/helm/configurations/values-dev.yaml
+++ b/helm/configurations/values-dev.yaml
@@ -23,3 +23,9 @@ components:
       image: reanahub/reana-message-broker
     reana_ui:
       image: reanahub/reana-ui
+
+quota:
+  disk_update: "*/2 * * * *"
+  termination_update_policy: "CPU"
+  # check new values
+  # periodic_update_policy: ""
+  # workflow_termination_update_policy: "cpu"
```

- Check that quotas are being calculated when configure by checking the CLI `reana-client quota-show` or the UI in the Profile page.
